### DISCUSSION TITLE
backup only hidden directories in the user's home directory

### DISCUSSION THREAD

**Discussion by user723045** [borgbase/vorta#1866] [FAQ]
Hello.
How can I get a similar pattern in Vorta as in the screenshot?
I've tried different options with exclude patterns but it doesn't work as it should.

I need to backup only hidden directories in the user's home directory.
TimeShift allows you to do this using the + and - attributes, but this does not work in Vorta.

![image](https://github.com/borgbase/vorta/assets/135850346/ffb5bbdd-11ca-4f86-af81-5c5058f7daac)

**Comment by real-yfprojects** [comment]
You can exclude all files/directories that do not start with `.` using the following pattern:

```
re:/home/kali/[^.].*
```
